data/net/minecraft/core/Vec3i.mapping

```diff
@@ -85,9 +104,16 @@ CLASS net/minecraft/core/Vec3i
 		ARG 1 y
 	METHOD setZ (I)Lnet/minecraft/core/Vec3i;
 		ARG 1 z
+	METHOD south ()Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i one unit south.
 	METHOD south (I)Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i south by the distance.
 		ARG 1 distance
 	METHOD subtract (Lnet/minecraft/core/Vec3i;)Lnet/minecraft/core/Vec3i;
-		ARG 1 vector
+		COMMENT Creates a new Vec3i subtracted by the subtrahend.
```

Comment:
This is technically incorrect, since if the vector is `(0, 0, 0)` then it returns the same vector object.